build/build-buildimage-bases.sh

```diff
@@ -33,6 +33,12 @@ echo "$PYTHON_IMAGE_PREFIX-3.6:$BUILD_NUMBER" >> $BUILD_IMAGE_BASES_ARTIFACTS_FI
 docker build -f $__REPO_DIR/images/build/python/3.7/Dockerfile -t "$PYTHON_IMAGE_PREFIX-3.7:$BUILD_NUMBER" $__REPO_DIR
 echo "$PYTHON_IMAGE_PREFIX-3.7:$BUILD_NUMBER" >> $BUILD_IMAGE_BASES_ARTIFACTS_FILE
 
+# Build Yarn cache
+YARN_CACHE_IMAGE_BASE="$ACR_DEV_NAME/public/oryx/build-yarn-cache"
+YARN_CACHE_IMAGE_NAME=$YARN_CACHE_IMAGE_BASE:$BUILD_NUMBER
+
+docker build $__REPO_DIR/images/build/yarn-cache -t $YARN_CACHE_IMAGE_NAME
```

Comment:
You probably need to provide the docker build context here (`$__REPO_DIR`) explicitly like the above docker build commands...Or else I think we could just `cd` into the repo root and can simplify all the commands in this files a bit

Reply:
I think it uses `images/build/yarn-cache` as the build context - which totally works - but is different from all of our other Dockerfiles

Reply:
@dorfire is correct. It is different because it is simpler, all we need is a bash script in there.

Reply:
I think the build context is the parameter that usually comes at the end: `Usage:  docker build [OPTIONS] PATH | URL | -`
Just to be clear, what i meant was something like below:
```
cd $__REPO_DIR

docker build -f images/build/python/3.7/Dockerfile -t "$PYTHON_IMAGE_PREFIX-3.7:$IMAGE_TAG" .

docker build -f images/build/yarn-cache/Dockerfile -t $YARN_CACHE_IMAGE_NAME .
```
Its the same consistent pattern

Reply:
Trust me, the command works. You can locally if you want to be sure

Reply:
There's no need to use -f here as the build context already has a dockerfile. It is the simplest possible docker build command.

Reply:
Sure, sounds good 👍